From 9f5d356954521100b05f93e9858eeca8087c6ec5 Mon Sep 17 00:00:00 2001
From: Konrad Leszczynski <konrad.leszczynski@intel.com>
Date: Tue, 8 Dec 2015 14:58:43 +0000
Subject: [PATCH 25/28] acpi: add new ACPI device for the SSIC sideband wakeup
 solution

This patch adds a new ACPI device (SSP0001) for the SSIC sideband
wakeup solution (Intel SSIC GPIO Driver). The driver will be
identified by this entry.

Change-Id: Ie61b4427153db504205291b08f8dee710482cfcf
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-9290
Signed-off-by: Konrad Leszczynski <konrad.leszczynski@intel.com>
Reviewed-on: https://android.intel.com:443/445722
---
 drivers/acpi/acpi_platform.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/acpi/acpi_platform.c b/drivers/acpi/acpi_platform.c
index 01e94cb..e25c11a 100644
--- a/drivers/acpi/acpi_platform.c
+++ b/drivers/acpi/acpi_platform.c
@@ -65,6 +65,7 @@ static const struct acpi_device_id acpi_platform_device_ids[] = {
 
 	{ "INT0002", 0 },
 	{ "HSP0001", 0 },
+	{ "SSP0001", 0 },
 
 	{ "INT33A2" },
 	{ "INTA4322" },
-- 
1.9.1

